### ISSUE TITLE
put on SD card like others previous LUA not WORKING

### ISSUE THREAD

**Issue by spacebubi** [codecae/betaflight-tx-lua-revised#6]
i'v paste from zip files after extracted, the 2 folders on the sd card of my 9XDplus....my previous BF.lua working if i put it on : /SCRIPTS/TELEMETRY and i paste also your folder outside that , marked BF . 

Nothing working...says "syntax error on line 7" and other strange ASCII

**Comment by codecae** [comment]
Looks like the installation is your issue.  There is a src and a sdcard directory.  The contents of the src directory does not need to be installed on your SD card,.. only the contents of the SD card needs to be used.  Copy the SCRIPTS directory under the sdcard folder to your SD instead of the entire contents of the github repository. https://github.com/codecae/betaflight-tx-lua-revised/tree/master/sdcard

**Comment by codecae** [comment]
if you copy the files over properly, it should work fine.

**Comment by codecae** [comment]
I updated the main repository readme to contain instructions for copying files to the SD card.

https://github.com/codecae/betaflight-tx-lua-revised

**Comment by spacebubi** [comment]
this is what i read on screen when try to lounch :

script error sintassi:
TELEMETRY/SPX9.lua :7
unexpected symbol near "<"

**Comment by spacebubi** [comment]
this is what i read:


syntax_error.jpg

**Comment by codecae** [comment]
It still looks like either you're starting the script wrong or you've put the wrong data on your SD card.  The files in the sdcard directory only have one line in each file, so it's impossible to have a syntax error on line 7.

**Comment by codecae** [comment]
Furthermore, when reviewing the source, there is no "<" on line 7.  Somehow the stuff on your sdcard is not in the right locations.

**Comment by umbertoghio** [comment]
OK, bothering you here instead of FB, here my setup and what I tried so far:
Taranis X9D SE with Crossfire Micro TX running OpenTX 2.2
Crossfire nano RX with Betaflight F3 FC on UART1, CRSF protocol and running 3.2 RC5
Started with a clean SD and installed sdcard-taranis-x9-2.2V0010.zip
Merged the Scripts content to my SD Scripts folder from the CRSF Branch
I run the Lua in the Telemetry folder, I see the usual Crossfire Lua page showing both TX and RX, but then get an "Unknown error" when selecting either.
I suspect I need to enable something in Betaflight, but I followed the related  issues up to 4120 without understanding if some requirement is not met.
I used to run the FrSky Lua scripts OK from the Smartport telemetry line on the X4R
Any clues? going to deep dive in the LUA code to find more
